### ISSUE TITLE
Can not install on centos7 because of libseccomp

### ISSUE THREAD

**Issue by Mabinogiysk** [docker/for-linux#320]
* [x] This is a bug report
* [ ] This is a feature request
* [x] I searched existing issues before opening this one

I do exactly what the docs says. When I execute the command ` yum install docker-ce`, I get an error:
```
--> Finished Dependency Resolution
Error: Package: docker-ce-18.03.1.ce-1.el7.centos.x86_64 (docker-ce-stable)
           Requires: libseccomp >= 2.3
           Available: libseccomp-2.2.1-1.el7.i686 (base)
               libseccomp = 2.2.1-1.el7
Error: Package: docker-ce-18.03.1.ce-1.el7.centos.x86_64 (docker-ce-stable)
           Requires: container-selinux >= 2.9
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
```
And there is no version of 2.3 in `yum list`

**Comment by thaJeztah** [comment]
What version of CentOS are you running? Can you post the output of `uname -a`?

**Comment by Mabinogiysk** [comment]
Sorry I forget it.
The output of `uname -a` is:
```
Linux c3916a7f-13f2-43c6-b3c9-3c4e130a685c 3.10.0-693.5.2.el7.x86_64 #1 SMP Fri Oct 20 20:32:50 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux
```

cat /etc/redhat-release 
```
CentOS Linux release 7.2.1511 (Core) 
```

**Comment by thaJeztah** [comment]
Yes, that's expected. Looks like you're running an obsolete version of CentOS 7; CentOS uses a rolling release model, which means that once CentOS 7.3 was released, CentOS 7.2 reached EOL; Older releases are not supported, and Docker won't be able to run on those.

From the [installation instructions](https://docs.docker.com/install/linux/docker-ce/centos/#os-requirements);

> ## OS requirements
>
> To install Docker CE, you need a maintained version of CentOS 7. Archived versions aren’t supported or tested.
>  The centos-extras repository must be enabled. This repository is enabled by default, but if you have disabled it, you need to re-enable it.

I'll close this issue because of the above, but let us know if you still run into issues on an up-to-date version of CentOS.
